### PR TITLE
bugfix: fix APISIX architecture image size distorted

### LINKED ISSUE #581: UI: APISIX architecture image size distorted
**Actual Behaviour**

image

**Steps to reproduce it**

Visit website. 

Some CSS mishap i guess.

### PR DESCRIPTION
**Pull request by leslie-tsang**
Fixes: #581 

Changes:
Remove width setting in customTheme.css

Screenshots of the change:
https://github.com/NGX-AI/apisix-website/blob/2424e3b009788cc46b24bf97f8fb65818272278c/website/src/css/customTheme.css#L62-L65

### CODE REVIEW

**Comment by 1502shivam-singh** [comment]
might want to update PR title to something like chore: or bug: to fix semantic pull req warning.

**Comment by leslie-tsang** [comment]
@1502shivam-singh updated

File: website/src/css/customTheme.css [hunk 1]
```diff
@@ -62,7 +62,6 @@ div.inner {
 .image {
   padding: 75px 10px 75px 75px;
   height: 200px;
-  width: 200px;
```
Comment (liuxiran) [inline website/src/css/customTheme.css:65]: Hi @leslie-tsang I noticed that only remove `width` is not enough, you can try to remove `height`, just let the image adapt to the current container

Comment (leslie-tsang) [reply website/src/css/customTheme.css:65]: @liuxiran  Roger that

**Comment by liuxiran** [comment]
also cc @Baoyuantop to have a look, thanks

**Review by liuxiran — APPROVED** [review]

**Review by Baoyuantop — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**